libavcodec/vvc/vvc_cabac.c

```diff
@@ -2201,17 +2260,34 @@ static inline int residual_coding_subblock(VVCLocalContext *lc, ResidualCoding *
         const int yc = (ys << rc->log2_sb_h) + rc->scan_y_off[n];
         const int *abs_level_pass1  = rc->abs_level_pass1 + yc * tb->tb_width + xc;
         int *abs_level              = rc->abs_level + yc * tb->tb_width + xc;
+        int abs_remainder;
 
         *abs_level = *abs_level_pass1;
-        if (abs_level_gt2_flag[n])
-            *abs_level += abs_remainder_decode(lc, rc, xc, yc) << 1;
+        if (abs_level_gt2_flag[n]) {
+            abs_remainder = abs_remainder_decode(lc, rc, xc, yc);
```

Comment:
int abs_reminder is better. 
It will tell the user, the abs_remainder was never used in the other places.